index.bs

```diff
@@ -816,62 +868,111 @@ To <dfn>handle funky elements</dfn> on a given |element|, run these steps:
-Sanitzer's operation based on it.
+A <dfn>sanitize action</dfn> can have the values `keep`, `drop`, or `block`.
 
-An effective configuration maps a given |element| or a given pair of
-|element| and |attribute| to a [=sanitize action=].
+<div algorithm="sanitize action for an element">
+To determine the <dfn>sanitize action for an |element|</dfn>, given a
+Sanitizer configuration dictionary |config|, run these steps:
 
-A <dfn>sanitize action</dfn> can have the values `keep`, `drop`, or `block`.
-To determine the <dfn>stricter action</dfn> of two [=sanitize actions=], pick
-the 'larger' of the two actions assuming a transitively defined order with
-`drop` &gt; `block`, and `block` &gt; `keep`.
+  1. Let |kind| be |element|'s [=element kind=].
+  1. If |element| is of `regular` |kind| and does not
+     [=element matches an element name|match=] any name in the
+     [=baseline element allow list=]: Return `drop`.
+  1. If |element| is of `custom` |kind| and if |config|'s
+     [=allow custom elements option=] is unset or set to anything other
+     than `true`: Return `drop`.
+  1. If |element| [=element matches an element name|matches=] any name
+     in |config|'s [=element drop list=]: Return `drop`.
+  1. If |element| [=element matches an element name|matches=] any name
+     in |config|'s [=element block list=]: Return `block`.
+  1. If |config| has a non-empty [=element allow list=] and
+     if |element| [=element matches an element name|matches=] any name
+     in |config|'s [=element allow list=]: Return `block`.
+  1. If |config| does not have a non-empty [=element allow list=] and
+     if |element| does not [=element matches an element name|match=] any name
+     in the [=default configuration=]'s [=element allow list=]: Return `block`.
+  1. Return `keep`.
+</div>
 
-<div algorithm="effective element configuration">
-To determine a Sanitizer |sanitizer|'s
-<dfn>effective element configuration</dfn> for an element |element|,
+<div algorithm="element matches an element name">
+To determine whether an <dfn>|element| matches an element |name|</dfn>,
 run these steps:
-  1. Let |config| be |sanitizer|'s [=configuration object=].
-  1. Let |baseline action| be the result of running the steps of the
-     [=determine the baseline configuration for an element=] algorithm
-     for the element |element|.
-  1. Let |config action| be the result of running the steps of the
-     [=determine the effective configuration for an element=] algorithm
-     for the element |element| and the config |config|.
-  1. Return the [=stricter action=] of |baseline action| and |config action|.
-
-Note: The definition of stricter actions ensures that the built-in baseline
-      configuration cannot be overriden, and therefor forms a hard guarantee
-      for all Sanitizer instances. Likewise for attributes.
+
+  1. Let |tokens| be the result of running the [=split on ascii whitespace=]
+     algorithm on |name|.
```

Comment:
Here you accept any ASCII whitespace. Above, you accept only U+0020. Is there a reason for one or the other?

Reply:
No, that was an oversight.

(Now resolved by using ":" instead.)